senza/components/auto_scaling_group.py

```diff
@@ -146,6 +146,9 @@ def component_auto_scaling_group(definition, configuration, args, info, force, a
         definition["Resources"][asg_name]["Properties"]["MaxSize"] = configuration["AutoScaling"]["Maximum"]
         definition["Resources"][asg_name]["Properties"]["MinSize"] = configuration["AutoScaling"]["Minimum"]
 
+        if configuration["AutoScaling"]["Minimum"] == 0:
+            definition["Resources"][asg_name]["Properties"]["DesiredCapacity"] = 1
```

Comment:
I would actually do:

```python
definition["Resources"][asg_name]["Properties"]["DesiredCapacity"] = max(configuration["AutoScaling"]["Minimum"], 1)
```

That would be more generic and less a "special" case (avoiding an "if" and avoiding code coverage decrease..)